### PR TITLE
Do not template config.py

### PR DESCRIPTION
**Pull request by jtojnar**
It breaks syntax highlighting in my editor. It should be enough to use a single file for variables.

### CODE REVIEW

File: src/api/python/speechd_config/paths.py.in [hunk 1]
```diff
@@ -2,3 +2,6 @@ SPD_CONF_ORIG_PATH="@spdconforigdir@"
 SPD_CONF_PATH="@spdconfdir@"
```
Comment (jtojnar) [inline src/api/python/speechd_config/paths.py.in:2]: Maybe I should also rename this file but I am not sure if it is not a part of public API. And also, these files are typically named `config.h` or `config.py` for Python but that name is taken.

Comment (sthibaul) [reply src/api/python/speechd_config/paths.py.in:2]: for safety, we can create another file, and make paths.py import it.

Comment (jtojnar) [reply src/api/python/speechd_config/paths.py.in:2]: Any ideas for name?

**Comment by sthibaul** [comment]
Thanks!

**Pull request merged**